newIDE/app/src/Utils/Serializer.js

```diff
@@ -8,7 +8,7 @@ const gd: libGDevelop = global.gd;
  * and unserializeFrom method.
  *
  * @param {*} serializable
- * @param {*} methodName The name of the serialization method. "unserializeFrom" by default
+ * @param {*} methodName The name of the serialization method. "serializeTo" by default
  */
```

Comment:
Probably worth removing the comment and add a better typing of the argument. 'serializeTo' | ...

Reply:
I'm not sure what to do because we probably don't want (or will forget) to update the type when a new function is needed in Core. I guess it's fine to  leave it as it is.